src/core.py

```diff
@@ -0,0 +1,14 @@
+import os
+import jinja2
+
+JINJA_ENVIRONMENT = jinja2.Environment(
+    loader=jinja2.FileSystemLoader(os.path.dirname(__file__) + "/../static/template"),
```

Comment:
You wouldn't have had to figure it out if you'd followed the project format this project started with.  I need to explain templates in general...I think in the readme.